recipes/python-codon-tables/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "python-codon-tables" %}
+{% set version = "0.1.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python_codon_tables-{{ version }}.tar.gz
+  sha256: 265beac928cbb77c6745bc728471adc7ffef933b794be303d272ecb9ad37d3d4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - python_codon_tables
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Edinburgh-Genome-Foundry/codon-usage-tables
+  summary: Codon Usage Tables for Python, from kazusa.or.jp
+  license: SAX-PD
```

```suggestion
  license: LicenseRef-PublicDomain
```
Since it doesn't have a license.

---

Do I need to add a license file for `LicenseRef-PublicDomain`?

---

No because there is none. `LicenseRef` is generally used for license types that are not in spdx.dev. In the end, core will have to decide what to do.

---

Never mind. License file was not necessary.